modules/end-user-guide/partials/proc_configuring-an-existing-workspace-to-use-intellij-idea.adoc

```diff
@@ -0,0 +1,15 @@
+[id="configuring-an-existing-workspace-to-use-intellij-idea_{context}"]
+= Configuring an existing workspace to use IntelliJ IDEA
+
+You can configure an already existing workspace to use IntelliJ IDEA as the editor in the {prod-short} dashboard.
+
+.Procedure
+
+. Navigate to the {prod-short} dashboard -> *Workspace* -> *Workspace Configuration* -> the *Editors*
+
+. Toggle the *IntelliJ IDEA Community Edition* switch.
++
+[NOTE]
+====
+If you have configured plugins or commands in your devfile when using Theia, they will not be accepted by IntelliJ IDEA since they were designed specifically for Theia.
```

Comment:
Recheck the usage of "since". Pg 25 in the IBM style guide: "Avoid terms that might be misinterpreted. For example, use once to mean one time, not to mean after
or when. Use since in relation to the passing of a period of time, not as a synonym for because."